AMQPPerfHarness/src/com/ibm/uk/hursley/perfharness/amqp/FloodPublisher.java

```diff
@@ -98,10 +121,42 @@ public boolean oneIteration() throws Exception {
 		else {
 			client.sendMessage("flood", topic, message);
 		}
+		messageRateControl ();
```

Comment:
Please see comment against the properties file relating to the existing -rt flag